AssemblerLib/MeshComponentMap.h

```diff
@@ -54,7 +54,7 @@ class MeshComponentMap
     MeshComponentMap getSubset(std::size_t const component_id,
                                MeshLib::MeshSubsets const& components) const;
 
-    /// The number of components in the map.
+    /// The number of dofs including the those located in the ghost nodes.
```

Comment:
In order to be consistent with the dof table, I'd suggest to rename
`size()` -> `dofSizeWithGhost«s»()`
`getNLocalUnknowns()` -> `dofSizeWithoutGhost«s»()`